test/Object/getOwnPropertyDescriptors.js

```diff
@@ -0,0 +1,89 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+var tests = [
+    {
+        name: "Object has getOwnPropertyDescriptors method",
+            body: function() {
+                assert.isTrue(Object.hasOwnProperty("getOwnPropertyDescriptors"), 'Object.hasOwnProperty("getOwnPropertyDescriptors")');
+
+            }
+    },
+    {
+        name: "Correctly handles bad parameters.",
+        body: function() {
+            assert.throws(function() {
+                WScript.Echo(Object.getOwnProperties());
+            }, TypeError, "");

```

Comment:
Please fill in the error message for the assertion to verify that we are seeing the expected error rather than some other TypeError.